### ISSUE TITLE
NPE in the new update

### ISSUE THREAD

**Issue by TrademarkTM** [bensku/Skript#378]
I was using the following code in the previous version without problems:
https://hastebin.com/latesexeqe.sql

However in the dev24 this error occour:
https://hastebin.com/ipayorajep.md

Version: dev24
Previous version: dev23
Server version: 1.11.2

**Comment by bensku** [comment]
This is not NullPointerException...

**Comment by TrademarkTM** [comment]
lol, sorry
There was many errors, and I forgot to paste the other ones ;P
https://hastebin.com/gusotuxiri.vbs

^^ all of them

**Comment by bensku** [comment]
Ah, these are indeed NPEs. Something has broken function unloading :(

**Comment by PugaBear** [comment]
@bensku I believe it has been slightly broken for a while, but dev24 made it worse. 

I had a command `/mgn` that would display when the next minigame night would be hosted. It used a function called getMonth that returned the name of the month (text) based on the numerical input. I noticed that this command always broke when the server restarted, but I never figured out why. Reloading the script would enable it again.

I just updated to dev24, and I noticed a _lot_ of errors in console. http://paste.bn-mc.net/irehu.vbs

I downgraded back to dev23, and found some more errors that I hadn't seen before, but not as many. http://paste.bn-mc.net/tuqos.vbs

As I was looking through, most of the errors involved scripts I had _just_ been working with and _knew_ they worked as expected when I was making them, so out of curiosity, I did `/sk reload arrowtrade` and it loaded fine. Same with the other scripts.
